Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -818,15 +818,44 @@ static int GetBonusOrPenaltyByEnemyType(DaggerfallEntity attacker, EnemyEntity A
             return damage;
         }
 
-        static int SavingThrow(DFCareer.EffectFlags effectFlags, DaggerfallEntity target)
+        static int SavingThrow(int effectType, DFCareer.EffectFlags effectFlags, DaggerfallEntity target, int modifier)
```

Comment:
If you can, would it be OK to make a small change here to rename effectType to elementType? We'll need to remap my usage of elements back to a classic enum or something to pass into here for type safety eventually.